tests/integration/test_main.py

```diff
@@ -1,4 +1,4 @@
-import pytest, jubilant, os
+import pytest, jubilant, os, json
```

Comment:
Linters would have opinions on this type of imports, and I tend to agree.

Widespread approach is to group imports with standard library, third party, local. Within the groups packages are typically kept on separate lines and some want them ordered alphabetically.